parity/cli.rs

```diff
@@ -286,6 +288,8 @@ pub struct Args {
 	pub cmd_import: bool,
 	pub cmd_signer: bool,
 	pub cmd_new_token: bool,
+  pub cmd_snapshot: bool,
+  pub cmd_restore: bool,
```

Comment:
it's because my editor automatically detects tabs vs spaces and we use spaces in the docopt string.
manually fixed now (hopefully)